### ISSUE TITLE
Support BTRS3 (bai3)

### ISSUE THREAD

**Issue by omerlh** [moov-io/bai2#117]
Hey, any chance you can support both version 2 and 3? I have BTRS (bai3) files that are parsed correctly by this CLI if I just change the header field. I am happy to open a PR allowing files with version either 2 or 3 in the first line...

**Comment by wokkaflokka** [comment]
I think a PR with a test that asserts behavior would be great!

**Comment by adamdecaf** [comment]
Do you have a doc with the Version 3 specification?

**Comment by omerlh** [comment]
Yes! At least I guess, this is what I got from the bank I am working with

**Comment by adamdecaf** [comment]
Follow-up from closing #118: BAI2 files with a version-3 header should use `IgnoreVersion` (v0.4.0+). Real BTR3/BAI3 needs its own parser; that work is next and will share the envelope scanner where the formats actually match.

**Comment by adamdecaf** [comment]
🤖 BAI3 / BTR3 is implemented in `pkg/bai3` with public X9 format-guide golden files under `test/testdata/bai3/`. `github.com/moov-io/bai2.Read` dispatches on the 01 version field (2 → `pkg/bai2`, 3 → `pkg/bai3`). `--ignoreVersion` still parses as BAI2 for mis-stamped files. See `docs/MIGRATING.md`.

**Comment by adamdecaf** [comment]
🤖 Landed on master (`c467733`).

- `pkg/bai3` is a separate BTR3 / BAI3 parser (X9.121 version 3)
- `github.com/moov-io/bai2.Read` dispatches on the 01 Version Number (`2` → `pkg/bai2`, `3` → `pkg/bai3`)
- `--ignoreVersion` / `IgnoreVersion` still means parse as BAI2 for mis-stamped files
- Goldens are public X9 format-guide samples under `test/testdata/bai3/` (the copyrighted spec PDF is not vendored)
- See `docs/MIGRATING.md`

**Comment by adamdecaf** [comment]
🤖 v1.0.0 is out: https://github.com/moov-io/bai2/releases/tag/v1.0.0
